internal/db/schema/migrations/postgres/11/01_server_type_enum.up.sql

```diff
@@ -0,0 +1,27 @@
+begin;
+
+create table server_type_enm (
+  name text primary key
+    constraint only_predefined_server_types_allowed
+      check (
+        name in (
+          'controller',
+          'worker'
+        )
+      )
+);
+comment on table server_type_enm is
+  'server_type_enm is an enumeration table for server types.'
```

Comment:
```suggestion
  'server_type_enm is an enumeration table for server types. '
```
Postgres automatically concatenates multi line comment strings but you need to add an extra space at the end of each line except the final line.